py/picca/delta_extraction/expected_fluxes/true_continuum.py

```diff
@@ -129,7 +129,8 @@ def _parse_config(self, config):
 
         self.num_processors = config.getint("num processors")
 
-        self.var_lss_binning = config.get("var lss binning", 'log')
+        self.raw_statistics_binning = config.get("raw statistics binning", 'log')
```

Comment:
This should be loaded via the `defaults` variable. This means adding between lines 22 and 23 a line with `"raw statistics binning" = "log"` and then replace this line with `self.raw_statistics_binning = config.get("raw statistics binning")`. This is necessary to include the default value in the .config.ini file that is  kept in the output folder